src/patternfly/demos/MasterDetail/examples/MasterDetail.css

```diff
@@ -0,0 +1,8 @@
+body .ws-example {
+  max-width: none;
+}
+
```

Comment:
do you need any of this anymore? Looks like it was needed when the examples weren't full screen